compiler/rustc_lint/src/foreign_modules.rs

```diff
@@ -5,19 +5,18 @@ use rustc_hir::def::DefKind;
 use rustc_middle::query::Providers;
 use rustc_middle::ty::layout::LayoutError;
 use rustc_middle::ty::{self, Instance, Ty, TyCtxt};
-use rustc_session::lint::{lint_array, LintArray};
 use rustc_span::{sym, Span, Symbol};
 use rustc_target::abi::FIRST_VARIANT;
 
 use crate::lints::{BuiltinClashingExtern, BuiltinClashingExternSub};
-use crate::types;
+use crate::{types, LintArray};
```

Comment:
last but not least, could you rename `LintArray` to `LintVec`? (if the type alias doesn't have many uses it might also make sense to remove it)

Reply:
It has a few uses, but I don't think it helps much since you'd still have to import `Lint`